test/Bridges/Constraint/semi_to_binary.jl

```diff
@@ -0,0 +1,120 @@
+using Test
+
+using MathOptInterface
+const MOI = MathOptInterface
+const MOIT = MathOptInterface.Test
+const MOIU = MathOptInterface.Utilities
+const MOIB = MathOptInterface.Bridges
+const MOIBC = MathOptInterface.Bridges.Constraint
+
+include("../utilities.jl")
+
+mock = MOIU.MockOptimizer(MOIU.UniversalFallback(MOIU.Model{Float64}()))
+config = MOIT.TestConfig()
+
+@testset "SemiToBinary" begin
+    bridged_mock = MOIBC.SemiToBinary{Float64}(mock)
+
+    bridge_type = MOIBC.SemiToBinaryBridge{Float64, MOI.Semiinteger{Float64}}
+    @test MOI.supports_constraint(bridge_type,
+                                  MOI.SingleVariable, MOI.Semiinteger{Float64})
+    @test MOIBC.concrete_bridge_type(bridge_type,
+                                     MOI.SingleVariable,
+                                     MOI.Semiinteger{Float64}) == bridge_type
+
+    @test MOI.supports(bridged_mock, MOI.ConstraintPrimalStart(), bridge_type)
+
+    MOIT.basic_constraint_tests(
+        bridged_mock, config,
+        include = [(F, S)
+                   for F in [MOI.SingleVariable]
+                   for S in [MOI.Semiinteger{Float64}, MOI.Semicontinuous{Float64}]])
+
+        MOIU.set_mock_optimize!(mock,
```

Comment:
Why is it indented ? It should be one level down like `MOIT.basic_constraint_tests`